static/app/components/events/eventReplay/replayContent.spec.tsx

```diff
@@ -72,29 +80,97 @@ const mockReplay: Partial<ReplayReader> = {
     },
     urls: ['http://localhost:3000/'],
     countUrls: 1,
-  }),
-  getRRWebEvents: () => [
+  },
+  rrwebEvents: [
+    {
+      type: 0,
+      data: {},
+      timestamp: 1663847686000,
+      delay: -198487,
+    },
+    {
+      type: 1,
+      data: {},
+      timestamp: 1663847749288,
+      delay: -135199,
+    },
     {
       type: 4,
       data: {
         href: 'http://localhost:3000/',
         width: 1536,
-        height: 824,
+        height: 722,
       },
-      timestamp: 1663025353247,
+      timestamp: 1663847749288,
+      delay: -135199,
     },
+  ],
+  breadcrumbs: [
     {
-      type: 4,
+      timestamp: 1663847751.886,
+      type: 'default',
+      category: 'ui.focus',
+    },
+    {
+      timestamp: 1663847765.355,
+      type: 'default',
+      category: 'ui.click',
+      message:
+        'input.form-control[type="text"][name="url"][title="Fully qualified URL prefixed with http or https"]',
       data: {
-        href: 'http://localhost:3000/',
-        width: 1536,
-        height: 151,
+        nodeId: 37,
+      },
+    },
+  ],
+  spans: [
+    {
+      op: 'navigation.navigate',
+      description: 'http://localhost:3000/',
+      startTimestamp: 1663847686.6915,
+      endTimestamp: 1663847749.2882,
+      data: {
+        size: 4731,
+        duration: 62615.89999997616,
+      },
+    },
+    {
+      op: 'navigation.navigate',
+      description: 'http://localhost:3000/',
+      startTimestamp: 1663847686.6915,
+      endTimestamp: 1663847749.2882,
+      data: {
+        size: 4731,
+        duration: 62615.89999997616,
       },
-      timestamp: 1663025450243,
     },
   ],
-  getDurationMs() {
-    return mockReplayDuration * 1000; // milliseconds
+  errors: [],
+  getDurationMs: function () {
+    return this.replayRecord.duration * 1000;
+  },
+
+  getReplay: function () {
+    return this.replayRecord;
+  },
+
+  getRRWebEvents: function () {
+    return this.rrwebEvents;
+  },
+
+  getRawCrumbs: function () {
+    return this.breadcrumbs;
+  },
+
+  getRawSpans: function () {
+    return this.spans;
+  },
+
+  isMemorySpan: function (span: ReplaySpan): span is MemorySpanType {
+    return span.op === 'memory';
+  },
```

Comment:
there are a lot of fields and methods inside this `mockReplay` object. It looks like it's supposed to match the ReplayReader class, but that class isn't even imported. 

What will happen to this test if the ReplayReader class changes? will it require changes, how will someone findout if it does, or doesn't?